java-sdk/connectors/feed-connector/src/main/java/com/lucidworks/fusion/connector/plugin/config/FeedConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.lucidworks.fusion.connector.plugin.config;
+
+import com.lucidworks.fusion.connector.plugin.api.config.ConnectorConfig;
+import com.lucidworks.fusion.connector.plugin.api.config.ConnectorPluginProperties;
+import com.lucidworks.fusion.schema.SchemaAnnotations.Property;
+import com.lucidworks.fusion.schema.SchemaAnnotations.RootSchema;
+import com.lucidworks.fusion.schema.SchemaAnnotations.StringSchema;
+
+@RootSchema(
+    name = "feed-random",
+    title = "Feed Random Connector",
+    description = "A feed-random connector",
+    category = "Feed"
+)
+public interface FeedConfig extends ConnectorConfig<FeedConfig.Properties> {
+
+  /**
+   * Connector specific settings
+   */
+  @Property(
+      title = "Properties",
+      required = true
+  )
+  Properties properties();
+
+  /**
+   * Connector specific settings
+   */
+  interface Properties extends ConnectorPluginProperties {
+
+    @Property(
+        title = "Feed file path",
+        description = "Feed file path location",
+        required = true,
+        order = 1
+    )
+    @StringSchema
+    String path();
```

Comment:
A local file doesn't work easily in a cluster  - could you change it to point to blobStore item or maybe to be able to it read from gist 'https://gist.github.com/mcondo/1186e69d8bbf2779c4db30d43a52850f'.

Reply:
@mcondo What were your thoughts on Cesar's recommendation?

Reply:
@kaitillman-lucidworks 
I made changes to have two options:
- Use `path` to locate a json file.
- Use some `generate properties`, so the connector will generate entries. You can see the details in the readme file I added -> https://github.com/lucidworks/connectors-sdk-resources/pull/49/files#diff-5df815cd1be027a310f519d96066e29eR8
and
https://github.com/lucidworks/connectors-sdk-resources/pull/49/files#diff-5df815cd1be027a310f519d96066e29eR26